eng/Signing.props

```diff
@@ -1,140 +1,181 @@
+        <!-- Sign the bundles, now that the engine is reattached. Avoid re-signing the engine. -->
+        <ItemsToSign
+          Include="@(BundleInstallerExeArtifact)"
+          Exclude="@(BundleInstallerEngineArtifact)" />
+        <!-- Note: wixstdba is internal to the engine bundle and does not get signed. -->
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignFinalPackages)' == 'true'">
+        <DownloadedSymbolPackages Include="$(DownloadDirectory)**\*.symbols.nupkg" />
+        <ItemsToSign Include="$(DownloadDirectory)**\*.nupkg" Exclude="@(DownloadedSymbolPackages)" />
+
+        <!-- The cross OS diagnostics symbol packages need to be signed as they are the only packages
+        that have a specific version of assets that are only meant to be indexed in symbol servers.
+        Since only *symbols.nupkg get indexed, and installer doesn't produce these, we need to glob them for signing. -->
+        <ItemsToSign Include="$(DownloadDirectory)**\*CrossOsDiag*.symbols.nupkg" />
+
+        <ItemsToSign Include="$(DownloadDirectory)**\*.deb" />
+        <ItemsToSign Include="$(DownloadDirectory)**\*.rpm" />
+      </ItemGroup>
+    </When>
+
+    <!-- When doing post build signing, we sign all artifacts we would push.
+         Symbol packages are included too. -->
+    <When Condition="'$(PostBuildSign)' == 'true'">
+      <ItemGroup>
+        <ItemsToSignWithPaths Include="$(DownloadDirectory)**/*.msi" Condition="'$(PrepareArtifacts)' == 'true'" />
+        <ItemsToSignWithPaths Include="$(DownloadDirectory)**/*.exe" Condition="'$(PrepareArtifacts)' == 'true'" />
+        <ItemsToSignWithPaths Include="$(DownloadDirectory)**/*.nupkg" Condition="'$(PrepareArtifacts)' == 'true'" />
+        <ItemsToSignWithPaths Include="$(DownloadDirectory)**/*.zip" Condition="'$(PrepareArtifacts)' == 'true'" />
+
+        <ItemsToSignWithoutPaths Include="@(ItemsToSignWithPaths->'%(Filename)%(Extension)')" />
+        <ItemsToSignPostBuild Include="@(ItemsToSignWithoutPaths->Distinct())" />
+      </ItemGroup>
+
+      <!-- Even when doing post build signing, sign mscordaccore*.dll and mscordbi.dll -->
+      <ItemGroup Condition="'$(SignBinaries)' == 'true'">
+        <ItemsToSign Include="$(CoreCLRSharedFrameworkDir)mscordaccore*.dll" />
+        <ItemsToSign Include="$(CoreCLRSharedFrameworkDir)mscordbi.dll" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignFinalPackages)' == 'true'">
+        <!-- The cross OS diagnostics symbol packages need to be signed as they are the only packages
+        that have a specific version of assets that are only meant to be indexed in symbol servers.
+        Since only *symbols.nupkg get indexed, and installer doesn't produce these, we need to glob them for signing. -->
+        <ItemsToSign Include="$(DownloadDirectory)**\*CrossOsDiag*.nupkg" />
+      </ItemGroup>
+    </When>
+  </Choose>
+
+  <ItemGroup>
+    <ItemsToSign Update="@(ItemsToSign)" Authenticode="$(CertificateId)" />
```

Comment:
Will this get an override by `FileSignInfo`?

Reply:
I actually don't think that does anything at all unless calling the Microbuild targets directly. I'll remove this and see what happens.

Reply:
Yeah, SignTool itself doesn't read this property, but it would be have been read by the Microbuild targets at one point.